array/DNAm/preprocessing/calcQCMetrics.r

```diff
@@ -201,6 +201,30 @@ if(!"pFilter" %in% colnames(QCmetrics)){
 	QCmetrics<-cbind(QCmetrics,"pFilter"= pFOut)
 }
 
+
+# detection p value and beacd count filtering for probe filtering
+if(!exists("probeFilt")){
+	# exclude really poor intensity, low bisulf conversion and failed pFilt samples
+	goodsamps <- c(QCmetrics$intensPASS & QCmetrics$bisulfCon > thresBS & QCmetrics$pFilter)
+
+	print("Running pfilter at probe level")
+	pFiltProbesPass<-apply.gdsn(node = pvals(gfile),
+	 margin = 1, FUN = function(x, y, z, passed) {
+				(sum(x[passed] > y, na.rm = TRUE)) < ((sum(!is.na(x)) * z)/100)
+			}, as.is = "logical", y = 0.05, z = 1, passed = goodsamps)

```

Comment:
Not really sure what these numbers actually mean. But it does what you want it to do.

Reply:
I see that it is very similar to:
https://github.com/ejh243/BrainFANS/blob/6bbfbeceb89f79421ad8e94d1c47bf00ba2010ee/array/DNAm/preprocessing/calcQCMetrics.r#L195-L198

So that probably explains where it came from (doesn't really tell me what the numbers are, but nothing to do with this PR).

Reply:
These are threshold values (percentages of probes and samples). I did wonder whether we wanted an option to define them in the config file. But then again I've only ever considering changing them once and it might be better not to over complicate the config file. An issue for future discussion/PR

Reply:
Okie dokes